web/src/components/article/edit/convert.ts

```diff
@@ -108,8 +108,9 @@ export function transformProsemirror(value: JsonValue): JsonValue {
       // Ensure `text` and get properties
       object.text = (object.content as [string] | undefined)?.[0] ?? ''
       delete object.content
-      object.programmingLanguage = (object.attrs as JsonObject)
+      object.programming_language = (object.attrs as JsonObject)
```

Comment:
I think we need a to apply this renaming from snake_case to camelCase, in one place, across all node types and in both directions. Otherwise, I suspect that this will crop again and again. Not sure where the best place for that is yet but will create an issue.